systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainST.java

```diff
@@ -643,7 +641,7 @@ void setUp(ExtensionContext extensionContext) {
 
         LOGGER.info("Setting producer and consumer properties");
 
-        oauthInternalClientJob = new KafkaOauthExampleClients.Builder()
+        new KafkaOauthExampleClients.Builder()
```

Comment:
I don't really understand why are you deleting this assignment, because now the builder is completely useless.

Reply:
it was not used at all.. Else I would not remove it. Seems like other (previous) change broke it. I am going to investigate what might have caused this.